research/object_detection/core/target_assigner.py

```diff
@@ -434,6 +436,9 @@ def create_target_assigner(reference, stage=None,
                                            use_matmul_gather=use_matmul_gather)
     box_coder_instance = faster_rcnn_box_coder.FasterRcnnBoxCoder()
 
+  elif reference == 'DETR':
```

Comment:
we don't need to add it here. In DETR meta architecture you can directly create `DETRTargetAssigner()`